src/App/Pages/Accounts/LoginSsoPageViewModel.cs

```diff
@@ -235,16 +230,51 @@ private async Task LogIn(string code, string codeVerifier, string orgId)
                         _syncService.FullSyncAsync(true).FireAndForget();
                         SsoAuthSuccessAction?.Invoke();
                     }
+                    else if (pendingRequest != null)
+                    {
+                        var authRequest = await _authService.GetPasswordlessLoginRequestByIdAsync(pendingRequest.Id);
+                        if (authRequest != null && authRequest.RequestApproved != null && authRequest.RequestApproved.Value)
+                        {
+                            var authResult = await _authService.LogInPasswordlessAsync(await _stateService.GetActiveUserEmailAsync(), authRequest.RequestAccessCode, pendingRequest.Id, pendingRequest.PrivateKey, authRequest.Key, authRequest.MasterPasswordHash);
```

Comment:
Does this also clear out the existing `pendingAdminAuthRequest`?

Reply:
it's in line 243 👍 `await _stateService.SetPendingAdminAuthRequestAsync(null);`